src/main/resources/view/DarkTheme.css

```diff
@@ -77,43 +77,50 @@
 }
 
 .split-pane:horizontal .split-pane-divider {
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: #121212;
     -fx-border-color: transparent transparent transparent #4d4d4d;
 }
 
 .split-pane {
     -fx-border-radius: 1;
     -fx-border-width: 1;
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: #121212;
 }
 
 .list-view {
-    -fx-background-insets: 0;
-    -fx-padding: 0;
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color:  #121212;
 }
 
 .list-cell {
     -fx-label-padding: 0 0 0 0;
     -fx-graphic-text-gap : 0;
-    -fx-padding: 0 0 0 0;
+    -fx-padding: 0px 0px 10px 0px;
+    -fx-background-color: #121212;
 }
 
-.list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+#cardDetailPane {
```

Comment:
Just a formatting thing, maybe this should be below since all the IDs seem to be grouped there?